assets/js/icons/index.js

```diff
@@ -0,0 +1,32 @@
+export { default as Icon } from './icon';
+
+export { default as bill } from './library/bill';
+export { default as card } from './library/card';
+export { default as cart } from './library/cart';
+export { default as comment } from './library/comment';
+export { default as discussion } from './library/discussion';
+export { default as exclamation } from './library/exclamation';
+export { default as external } from './library/external';
+export { default as folderStared } from './library/folder-stared';
+export { default as folder } from './library/folder';
+export { default as image } from './library/image';
+export { default as grid } from './library/grid';
+export { default as list } from './library/list';
+export { default as more } from './library/more';
+export { default as notes } from './library/notes';
+export { default as notice } from './library/notice';
+export { default as tags } from './library/tags';
+export { default as radioSelected } from './library/radio-selected';
+export { default as radioUnselected } from './library/radio-unselected';
+export { default as review } from './library/review';
+export { default as search } from './library/search';
+export { default as server } from './library/server';
+export { default as star } from './library/star';
+export { default as stonks } from './library/stonks';
+export { default as tag } from './library/tag';
+export { default as thumbUp } from './library/thumb-up';
+export { default as toggle } from './library/toggle';
+export { default as trash } from './library/trash';
+export { default as truck } from './library/truck';
+export { default as widgets } from './library/widgets';
+export { default as woo } from './library/woo';
```

Comment:
Nitpicking, but in case we are ordering them alphabetically, `image` should go after `grid` and `tags` after `tag`.